.github/workflows/deploy-on-release.yml

```diff
@@ -0,0 +1,633 @@
+name: libcellml-deploy-on-release
+
+on:
+  release:
+    types:
+      - created
+
+env:
+  BUILD_TYPE: Release
+  MACOSX_DEPLOYMENT_TARGET: 10.14
+
+jobs:
+  many-linux-wheels:
+    name: ${{ matrix.config.name }}
+    runs-on: ubuntu-latest
+    container: quay.io/pypa/manylinux2014_x86_64
```

Comment:
I guess you are aware of the message at https://quay.io/pypa/manylinux2014_x86_64, i.e.
> On June 30 2021, Quay.io will move to Red Hat Single Sign-On Services. If you haven't done so already, please [create a PERSONAL (not corporate) Red Hat account](https://www.redhat.com/wapps/ugc/register.html) and attach it to your Quay.io account, following [these instructions](https://access.redhat.com/articles/5363231). If you create a corporate account you WILL lose access to your Quay personal namespace